mtp_api/apps/transaction/tests/test_bank_admin_views.py

```diff
@@ -82,3 +85,294 @@ def test_create_list(self):
             ).count(),
             len(data_list)
         )
+
+    def _test_create_list_incomplete_data(self, null_field):
+        url = self._get_url()
+        data_list = self._get_transactions_data()
+
+        user = self.bank_admins[0]
+        data_list[0][null_field] = None
+
+        current_count = Transaction.objects.count()
+
+        response = self.client.post(
+            url, data=data_list, format='json',
+            HTTP_AUTHORIZATION=self.get_http_authorization_for_user(user)
+        )
+        self.assertEqual(response.status_code, http_status.HTTP_400_BAD_REQUEST)
+
+        # check no change in db
+        self.assertEqual(current_count, Transaction.objects.count())
+
+    def test_create_list_null_account_number_bad_request(self):
+        self._test_create_list_incomplete_data('sender_account_number')
+
+    def test_create_list_null_sort_code_bad_request(self):
+        self._test_create_list_incomplete_data('sender_sort_code')
+
+    def test_create_list_null_amount(self):
+        self._test_create_list_incomplete_data('amount')
```

Comment:
Renaming is fine.

What is your rationale for increasing code duplication in this case? I can see the reasoning behind e.g. not making a test just a list of helper methods, because then the test logic is scattered in multiple locations, but this is really just a single test that happens to be being run with different inputs. If you want to know what happens in the test case you only have one place to look.